### DISCUSSION TITLE
No map component available on self-hosted installation?

### DISCUSSION THREAD

**Discussion by offbase-solutions** [illacloud/illa-builder#2704] [General]
Why is there no map component available in the self-hosted, is this a paid feature. Important would like to know.

**Comment by donqu1xotechina** [comment]
Hey There:
Due to the Google Requirement, we won't be able to provide this feature in self host version. So this is only available on ILLA Cloud. You can use free version to try it out if you like. Free feel to join our Discord for more support.
